bulla-contracts/schema.graphql

```diff
@@ -137,6 +137,30 @@ type ClaimCreatedEvent implements IClaimEvent & IEventLog @entity {
   timestamp: BigInt!
 }
 
+type FinancingOfferedEvent implements IEventLog @entity {
+  id: ID!
+  originatingClaimId: Claim!
+  minDownPaymentBPS: Int!
+  interestBPS: Int!
+  termLength: BigInt!
+  eventName: String!
+  blockNumber: BigInt!
+  transactionHash: Bytes!
+  logIndex: BigInt!
+  timestamp: BigInt!
+}
+
+type FinancingAcceptedEvent implements IEventLog @entity {
+  id: ID!
+  originatingClaimId: Claim!
+  financedClaimId: Claim!
```

Comment:
Do we need financedClaimId? Wouldn't this be the id?

Reply:
I see that the event Id is a string that is built, you can probably disregard this